packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -139,21 +147,8 @@ class CupertinoTextSelectionToolbar extends StatelessWidget {
       return outputChild;
     }
     return DecoratedBox(
-      // These shadow values were eyeballed from a screenshot of iOS 16.3.1, as
-      // light mode didn't appear in the Apple design resources assets linked at
-      // the top of this file.
-      decoration: BoxDecoration(
-        borderRadius: const BorderRadius.all(_kToolbarBorderRadius),
-        boxShadow: <BoxShadow>[
-          BoxShadow(
-            color: CupertinoColors.black.withOpacity(0.2),
-            blurRadius: 15.0,
-            offset: Offset(
-              0.0,
-              isAbove ? 0.0 : _kToolbarArrowSize.height,
-            ),
-          ),
-        ],
+      decoration: const BoxDecoration(
+        borderRadius: BorderRadius.all(_kToolbarBorderRadius),
```

Comment:
Do we still need the border radius here? It will be clipped after the toolbar is painted right?

Reply:
Yeah, this can go - we're not drawing any borders here.